aws/resource_aws_gamelift_fleet_test.go

```diff
@@ -556,6 +558,7 @@ resource "aws_gamelift_fleet" "test" {
   name              = "%s"
   description       = "%s"
   instance_role_arn = "${aws_iam_role.test.arn}"
+  fleet_type        = "ON_DEMAND"
```

Comment:
We should ensure existing configurations do not show a difference by removing these additions to the test configurations. The `resource.TestCheckResourceAttr()` additions above are okay. 👍